lib/subjects/remover.rb

```diff
@@ -0,0 +1,62 @@
+module Subjects
```

Comment:
Not really - I've always had trouble figuring out how best to organise  `/lib` in a rails app.  I was keeping it in line with subject selection work done earlier which was moved to lib, which in my mind was doing work with the models instead of being concerned, etc. If you like we can move them into app somewhere....